src/structures/Guild.js

```diff
@@ -736,13 +743,44 @@ class Guild extends Base {
    *   .catch(console.error);
    */
   fetchVanityCode() {
+    return require('util').deprecate(() => {
+      if (!this.features.includes('VANITY_URL')) {
+        return Promise.reject(new Error('VANITY_URL'));
+      }
+      return this.client.api
+        .guilds(this.id, 'vanity-url')
+        .get()
+        .then(res => res.code);
+    }, 'fetchVanityCode() is deprecated. Use fetchVanityData() instead.');
+  }
+
+  /**
+   * An object containing information about a guild member's ban.
```

Comment:
this should be changed, its definitely not information about a guild member's ban

Reply:
Lol oops, I overlooked that